PopupBridge/src/test/java/com/braintreepayments/api/PopupBridgeClientUnitTest.kt

```diff
@@ -226,19 +226,22 @@ class PopupBridgeClientUnitTest {
     fun `when handleReturnToApp is called and browser switch fails, canceled javascript is run`() = runTest {
```

Comment:
Can we update the test name to call out the error instead of the canceled javascript?